dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/LoggingConstants.java

```diff
@@ -71,7 +71,7 @@ public static enum TraceFormat {
 
     // Stack Joiner feature configurations
     String ENV_WLP_LOGGING_MAX_SYSTEM_STREAM_PRINT_EVENT_SIZE = "WLP_LOGGING_MAX_SYSTEM_STREAM_PRINT_EVENT_SIZE";
-    String ENV_WLP_LOGGING_STACK_JOIN = "WLP_LOGGING_STACK_JOIN";
+    String ENV_WLP_LOGGING_STACK_JOIN = "WLP_LOGGING_STACK_TRACE_SINGLE_ENTRY";
```

Comment:
Will you also rename `ENV_WLP_LOGGING_STACK_JOIN` to `ENV_WLP_LOGGING_STACK_TRACE_SINGLE_ENTRY`?

Reply:
fixed

Reply:
Have you pushed the changes yet?